nuitka/plugins/PluginBase.py

```diff
@@ -910,3 +914,28 @@ def wrapped(*args, **kwargs):
                 return None
 
     return wrapped
+
+
+class TagContext(dict):
+    def __init__(self, full_name, *args, **kwargs):
+        self.full_name = full_name
+        super().__init__(*args, **kwargs)
+
+    def __getitem__(self, key):
+        try:
+            dict.__getitem__(self, key)
+
+        except KeyError:
+            if key.startswith("use_"):
+                Tracing.plugins_logger.info(
+                    "Use default 'False' for not existing tag '" + key + "'."
+                )
+                return False
+
+            Tracing.plugins_logger.sysexit(
+                "Tag '"
```

Comment:
Please use `%` format, it is so much more readable. `"Tag '%s' in module ..." % (tag, ...)` is what all of Nuitka uses. I played with `.format` too, and would love f-strings, but it's a good compromise in my book for not too large messages.